llama_hub/llama_packs/multidoc_autoretrieval/multidoc_autoretrieval.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        vector_store_info,
</pre><p>looks like we haven't defined this yet? Did we want to copy it from the "core" docs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama-hub/pull/803/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>good catch </p>